nixos/modules/services/x11/gdnc.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.gdnc;
+in {
+  options = {
+    services.gdnc.enable = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = "Enable gdnc, the GNUstep distributed notification center";
+    };
+  };
+  config = mkIf cfg.enable {
+    assertions = singleton {
+      assertion = config.services.gdnc.enable -> config.services.gdomap.enable;
+      message = "Cannot start gdnc without starting gdomap";
+    };
+    environment.systemPackages = [ pkgs.gnustep_make pkgs.gnustep_base ];
+    systemd.services.gdnc = {
+      path = [ pkgs.gnustep_base ];
+      description = "gdnc: GNUstep distributed notification center";
+      requires = [ "gdomap.service" ];
```

Comment:
Possibly should also have an ordering directive; dependencies without ordering implies parallel activation.